tests/_pom/pages/commentstimelinepage.js

```diff
@@ -27,12 +27,20 @@ class CommentsTimelinePage extends GitHubPage {
 	 */
 	async editComment( index ) {
 		const root = ( await this.browserPage.$$( 'form.js-comment-update' ) )[ index ];
-		const editButton = await root.evaluateHandle( root =>
-			root.closest( '.timeline-comment' ).querySelector( '.js-comment-edit-button' ) );
-		const actionButton = await editButton.evaluateHandle( editButton =>
-			editButton.closest( 'details-menu' ).previousElementSibling );
+
+		const actionButton = await root.evaluateHandle( root =>
+			root.closest( '.timeline-comment' ).querySelector( '.show-more-popover' )
+				.parentElement.querySelector( 'summary[role=button]' ) );
+
+		actionButton.hover();
+		await this.browserPage.waitFor( 2000 );
```

Comment:
We must at all cost avoid fixed-time timeouts like this (what if the code takes just a little more than 2 secs?). Instead we should wait for an element to be visible / present - this way the code resumes as quickly as possible and waits as long as needed when things are working slow. That's much greater reliability.

If this is really not possible you have to explain why exactly this is impossible.

I see there's one more fixed timeout in this PR to be addressed.